docs/Using-Virtual-Environment.md

```diff
@@ -1,39 +1,52 @@
 # Using Virtual Environment
 
 ## What is a Virtual Environment?
-A Virtual Environment is a self contained directory tree that contains a Python installation
-for a particular version of Python, plus a number of additional packages. To learn more about
-Virtual Environments see [here](https://docs.python.org/3/library/venv.html).
+
+A Virtual Environment is a self contained directory tree that contains a Python
```

Comment:
@mmattar are you setting a new line width for md files? If yes, how do we ensure that the same convention is followed in all md files?